bot/exts/easter/save_the_planet.py

```diff
@@ -0,0 +1,28 @@
+import json
+import random
+from pathlib import Path
+
+from discord import Embed
+from discord.ext import commands
+
+
+class SaveThePlanet(commands.Cog):
+    """A cog that teaches users how they can help our planet."""
+
+    json_embeds = []
+
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot = bot
+        with open(Path("bot/resources/save_the_planet.json"), 'r', encoding='utf8') as f:
```

Comment:
```suggestion
        with open(Path("bot/resources/easter/save_the_planet.json"), 'r', encoding='utf8') as f:
```
Seems like the wrong path is used.

Reply:
OK, I'll fix that.